### ISSUE TITLE
HIde read receipts does not work for audio

### ISSUE THREAD

**Issue by jsalatiel** [suraj0208/WhatsappExtensions#9]
Would it be possible to make hide read receipts for audio messages ?

**Comment by suraj0208** [comment]
are you sure other person gets blue ticks ? I tried it and it was perfectly working for me.

**Comment by jsalatiel** [comment]
Yes. Just tested here and the other person gets the blueticket right after i listen the audio message. Strange.

**Comment by suraj0208** [comment]
yeah. The receipt is sent after you listen the audio. my bad. I'll see what I can do. Thanks for reporting this issue.
